apps/src/panels/types.ts

```diff
@@ -1,12 +1,15 @@
 // The level data for a panels level that doesn't require
 // reloads between levels.
 export interface PanelsLevelData {
-  panels: [
-    {
-      imageUrl: string;
-      text: string;
-      nextUrl?: string;
-      layout?: string;
-    }
-  ];
+  panels: Panel[];
 }
+
+export interface Panel {
+  imageUrl: string;
+  text: string;
+  nextUrl?: string;
+  layout?: PanelLayout;
+  key: string;
+}
+
+export type PanelLayout = 'text-bottom-left' | 'text-top-right';
```

Comment:
nit: declare this type before declaring `Panel` mybe? So that it'll be easier to read types